source/adapters/opencl/command_buffer.hpp

```diff
@@ -46,6 +46,8 @@ struct ur_exp_command_buffer_handle_t_ {
   /// Set to true if the kernel commands in the command-buffer can be updated,
   /// false otherwise
   bool IsUpdatable;
+  /// Set to true if the command-buffer was created from an in-order queue.
+  bool IsInOrder;
```

Comment:
```suggestion
  /// Set to true if the command-buffer was created with an in-order queue.
  bool IsInOrder;
```
Extreme nit but I think `with` describes the relationship better than `from`.